pkg/osquery/extension.go

```diff
@@ -304,6 +305,106 @@ func isNodeInvalidErr(err error) bool {
 	}
 }
 
+// addRegistration should be called after enrollment to store enrollment/registration details
+// in our persistent store under e.registrationId. We frequently use the munemo associated with
+// the registration (e.g. in checkups), so we parse the enrollment secret in order to extract
+// the munemo from the claims and store it as well.
+func (e *Extension) addRegistration(nodeKey string, enrollmentSecret string) error {
```

Comment:
how gnarly would it be to to add some assertions to an enrollment test to make sure these get stored?

Reply:
I think not bad, actually! Let me see 👀

Reply:
Not bad at all! Updated, thank you for the suggestion!